docker/chromium/Dockerfile

```diff
@@ -4,6 +4,7 @@ FROM demisto/python3-deb:3.10.13.87666
 
 COPY requirements.txt .
 
+ARG POPPLER_VERSION=24.02.0
```

Comment:
is there a way to query their repo for the latest version so we arent stuck on a specific version? or do we want to lock here?